pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -100,6 +101,14 @@ func (c *Controller) GetAppliedToGroupNum() int {
 	return c.ruleCache.GetAppliedToGroupNum()
 }
 
+func (c *Controller) GetNetworkPolicyControllerInfo() ClusterinformationV1beta1.NetworkPolicyControllerInfo {
```

Comment:
Can we avoid adding duplicate functions to export same information? This and `GetAppliedToGroupNum`, `GetAddressGroupNum` and `GetNetworkPolicyNum`. Ideally we can avoid to let NetworkPolicyController know the struct of ClusterinformationV1beta1.NetworkPolicyControllerInfo.

Reply:
I moved GetNetworkPolicyControllerInfo() to interface AgentQuerier. Controller side is the same.